packages/flutter_tools/lib/src/ios/xcresult.dart

```diff
@@ -40,30 +40,37 @@ class XCResultGenerator {
     List<XCResultIssueDiscarder> issueDiscarders = const <XCResultIssueDiscarder>[],
   }) async {
     final Version? xcodeVersion = xcode.currentVersion;
-    final RunResult result = await processUtils.run(<String>[
-      ...xcode.xcrunCommand(),
-      'xcresulttool',
-      'get',
-      // See https://github.com/flutter/flutter/issues/151502
-      if (xcodeVersion != null && xcodeVersion >= Version(16, 0, 0)) '--legacy',
-      '--path',
-      resultPath,
-      '--format',
-      'json',
-    ]);
+    final bool useNewCommand = xcodeVersion != null && xcodeVersion >= Version(16, 0, 0);
+
+    // Base command for xcrun
```

Comment:
I feel like a lot of these comments are a little bit extraneous or too obvious. For example, if you named have `command` be named `baseCommand` and you could skip this comment. Could you go through this pr and clean up the comments a little?

Reply:
done

Reply:
done